src/SearchResults.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import CostomerProfile from "./CustomerProfile";
+import moment from "moment";
+
+const SearchResults = props => {
+  const [selectedId, setselectedId] = useState(undefined);
+  const [customerProfile, setCustomerProfile] = useState("");
+
+  const rowHighlight = bookingId => {
+    setselectedId(bookingId);
+  };
+
+  const profilehandler = bookingId => {
+    setCustomerProfile(bookingId);
+  };
+
+  return (
+    <>
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="col">id</th>
+            <th scope="col">title</th>
+            <th scope="col">first name</th>
+            <th scope="col">surname</th>
+            <th scope="col">email</th>
+            <th scope="col">room id</th>
+            <th scope="col">check in date</th>
+            <th scope="col">check out date</th>
+            <th scope="col">nights stayed</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.data.map(person => {
+            let a = person.checkOutDate;
+            let b = person.checkInDate;
```

Comment:
Maybe we could use `const` for these variables? It doesn't look like they change